usr/lib/systemd/user/regolith-x11.service

```diff
@@ -0,0 +1,41 @@
+[Unit]
+Description=Regolith Session on i3
+# On X11, try to show the GNOME Session Failed screen
+OnFailure=gnome-session-failed.target
+OnFailureJobMode=replace
+CollectMode=inactive-or-failed
+RefuseManualStart=on
+RefuseManualStop=on
+
+After=gnome-session-manager.target
+
+Requisite=gnome-session-initialized.target
+PartOf=gnome-session-initialized.target
+Before=gnome-session-initialized.target
+
+ConditionEnvironment=XDG_SESSION_TYPE=x11
+
+# Limit startup frequency more than the default
+StartLimitIntervalSec=15s
+StartLimitBurst=3
+
+[Service]
+Slice=session.slice
+Type=notify
+NotifyAccess=all
+ExecStart=regolith-session-x11
+
+# Setup custom environment for session
+EnvironmentFile=-%h/.config/regolith3/sway/env
```

Comment:
Should this be sway?

Reply:
Thanks for spotting copypasta. You're correct.